src/main/java/mekanism/common/integration/multipart/MultipartTransmitter.java

```diff
@@ -28,6 +30,28 @@ public IPartSlot getSlotFromWorld(IBlockAccess world, BlockPos pos, IBlockState
 	}
 	
 	@Override
+	public void onAdded(IPartInfo part)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			((TileEntitySidedPipe)tile).onPartChanged(null);
+		}
+	}
+	
+	@Override
+	public void onPartAdded(IPartInfo part, IPartInfo otherPart)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			tile.validate();
```

Comment:
`validate` is used here not to trigger `onWorldJoin` but instead to make sure the tile entity remains valid when a multipart is added around an existing transmitter. If `validate` is not called at this point then the transmitter gets removed as invalid but since the TileEntity remains invalid the `OrphanPathFinder` doesn't iterate over it and it remains permanently orphaned.